index.html

```diff
@@ -0,0 +1,128 @@
+					<p>1: $2.00</p>
+					<p>6: $10.00</p>
+					<p>Baker's Dozen: $20.00</p>
+				</section>
+			</article>
+		</div>
+
+		<!-- PRODUCT 2 ARTICLE -->
+
+		<div class="card">
+			<article>
+				<header class="card-header">
+					<h1>Fancy Cupcake</h1>
+				</header>
+				<section>
+					<img src="https://farm2.staticflickr.com/1634/26221206642_0db7b73e7c_z.jpg" alt="Cupcakes for Class Project">
+					<div class="card-description">
+						<p>Cotton candy halvah liquorice. Ice cream sweet pudding icing jelly chupa chups toffee cake. Oat cake jelly-o apple pie lemon drops I love. Macaroon apple pie cheesecake gingerbread carrot cake brownie I love I love. Cupcake halvah cake gingerbread pastry carrot cake gummi bears. Jujubes oat cake pudding icing I love I love carrot cake. Tootsie roll donut danish.</p>
+					</div>
+					<div class="availability">
+						<p>Available</p>
+					</div>
+				</section>
+				<section class="specs">
+					<header>
+						<h3>Specifications:</h3>
+					</header>
+					<p>Size: 5"x5"</p><!-- specifications on size -->
+					<p>Weight: 4 oz.</p><!-- specifications on weight -->
+					<footer>
+						<h4>Specifications become invalid at end-of-business today.</h4>
+					</footer>
+				</section>
+				<section class="specs">
+					<header>
+						<h3>Pricing:</h3>
+					</header>
+					<!-- three block elements that contain information about pricing -->
+					<p>1: $2.50</p>
+					<p>6: $14.00</p>
+					<p>Baker's Dozen: $28.00</p>
+				</section>
+			</article>
+		</div>
+
+		<!-- PRODUCT 3 ARTICLE -->
+
+		<div class="card">
+			<article>
+				<header class="card-header">
+					<h1>Artisanal Cupcake</h1>
+				</header>
+				<section>
+					<img src="https://farm2.staticflickr.com/1623/25708748894_8474718fd1.jpg" alt="Cupcakes for Class Project">
+					<div class="card-description">
+						<p>Macaroon chupa chups oat cake lemon drops jujubes bear claw cupcake icing lollipop. Soufflé pastry cupcake donut pie liquorice cotton candy I love. Tootsie roll dessert. I love tiramisu toffee marshmallow I love biscuit I love liquorice cheesecake. Croissant donut muffin. Powder chupa chups cake I love carrot cake. Toffee pudding halvah wafer tootsie roll.</p>
+					</div>
+					<div class="availability">
+						<p>Not Available</p>
+					</div>
+				</section>
+				<section class="specs">
+					<header>
+						<h3>Specifications:</h3>
+					</header>
+					<p>Size: 6"x6"</p><!-- specifications on size -->
+					<p>Weight: 5 oz.</p><!-- specifications on weight -->
+					<footer>
+						<h4>Specifications become invalid at end-of-business today.</h4>
+					</footer>
+				</section>
+				<section class="specs">
+					<header>
+						<h3>Pricing:</h3>
+					</header>
+					<!-- three block elements that contain information about pricing -->
+					<p>1: $12.79</p>
+					<p>6: $84.23</p>
+					<p>Baker's Dozen: $193.56</p>
+				</section>
+			</article>
+		</div>
+	</div>
+</body>
+</html>
```

Comment:
As Github helpfully points out with the 🚫  icon, it's convention from þe olde POSIX days to end all files with a blank new line.  You can read more about it [here](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) if you're interested.